EasyCommands.Tests/ScriptTests/GyroscopeBlockTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using Moq;
+using Sandbox.ModAPI.Ingame;
+using VRageMath;
+
+namespace EasyCommands.Tests.ScriptTests {
+    [TestClass]
+    public class GyroscopeBlockTests {
+        [TestMethod]
+        public void EnableGyroscopeOverrides() {
+            String script = @"
+set the ""test gyro"" auto to false
```

Comment:
maaaayyyybeee think about just calling it override and having it map 1:1 to the boolean.

Reply:
did it